components/StyledCollectiveCard.js

```diff
@@ -154,12 +154,12 @@ const StyledCollectiveCard = ({
         width="95%"
         right="0"
         pt="41.25%"
-        style={{ background: getBackground(collective) }}
+        style={{ background: getBackground(collective), filter: collective.isFrozen ? 'grayscale(1)' : undefined }}
```

Comment:
I was hesitant to apply it there because it would grey scale all the card's children, and there are some buttons on the host admin page that are still active even if the collective is frozen.

Reply:
Makes sense!